### ISSUE TITLE
missing documentation about aria-label empty by default

### ISSUE THREAD

**Issue by rkristelijn** [angular-slider/ngx-slider#272]
Because of inclusive reasons and because it is kind of enforced by law, web solutions should be accessible. An inaccessible solution is a broken solution. 

Digging though the codebase I found that `ariaLabel` is a property of the `options` that can be passed to `<ngx-slider` however I think things should be accessible by default, so an empty aria-label is not as it should be.

*reproducing*
Go to one of the demo's:

https://ngx-slider-simple-slider-example.stackblitz.io/

run axe-core 4.2.3 on it

![image](https://user-images.githubusercontent.com/21142148/125913779-38defc84-00d3-40f0-aa35-c6c644c3470b.png)

rules: https://dequeuniversity.com/rules/axe/4.2/aria-input-field-name?application=AxeChrome

problem: `aria-label` empty

![image](https://user-images.githubusercontent.com/21142148/125913950-667e0ff5-ccd6-451b-bafc-57cc486ba067.png)

related to #146

**Comment by rkristelijn** [comment]
@piotrdz is this repo still maintained?

**Comment by Makatumba** [comment]
The `aria-label` can be used without changes in your fork.
It is possible to Provide the  `ariaLabel` in `Option` Object. 

@rkristelijn thx for your fork. Without it I wouldn't be abler to find this solution.

**Comment by mjknight50** [comment]
I haven't figured out how to fix this. 

No matter what, I get an empty `aria-labelledby`
